### DISCUSSION TITLE
CSV file

### DISCUSSION THREAD

**Discussion by Sarali97** [pastas/pastas#284] [Q&A] [answered]
Hello!

First of all, thanks for a great model it has been of much help!

I have one question though, is there any way to extract the data for the simulated time-series into a CSV file. 

Best regards,
Sarah :)

**Comment by dbrakenhoff** [comment, marked as answer]
Hi Sarah,

Check out [Pandas documentation](https://pandas.pydata.org/docs/reference/api/pandas.Series.to_csv.html) about how to export pandas Series or DataFrames to csv! But this should work:

```python
sim = ml.simulate(tmin=tmin, tmax=tmax)
sim.to_csv("simulation.csv")
```
